### ISSUE TITLE
Dark Background when using rounded corners on the Custom Dialog popup

### ISSUE THREAD

**Issue by DarckETH** [FlutterFlow/flutterflow-issues#2147]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

Whenever I use rounded corners for my main container and use that component as a custom dialog. It shows a dark background in Safari on Mac where there is no background of the container due to the rounded corners.

![Scherm­afbeelding 2024-01-12 om 09 17 49](https://uploads.linear.app/3241089d-736a-4422-900b-051e72accdbe/02f3f2b5-2735-4915-9a47-31015442809c/61e6acfd-e081-4362-9576-64aee36c1b3d?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMyNDEwODlkLTczNmEtNDQyMi05MDBiLTA1MWU3MmFjY2RiZS8wMmYzZjJiNS0yNzM1LTQ5MTUtOWE0Ny0zMTAxNTQ0MjgwOWMvNjFlNmFjZmQtZTA4MS00MzYyLTk1NzYtNjRhZWUzNmMxYjNkIiwiaWF0IjoxNzA1NzM2NjcxLCJleHAiOjMzMjc2Mjk2NjcxfQ.z3Uw_0ygfxkgdtQ6paQymq3Xuvxc_PMkxV0rpNEVMKs "=143x")

![Scherm­afbeelding 2024-01-12 om 09 17 40](https://uploads.linear.app/3241089d-736a-4422-900b-051e72accdbe/d68f4b9b-378a-4402-b04f-beaf619cdd3c/e83b2a91-93d4-423a-9eab-488c644f577b?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMyNDEwODlkLTczNmEtNDQyMi05MDBiLTA1MWU3MmFjY2RiZS9kNjhmNGI5Yi0zNzhhLTQ0MDItYjA0Zi1iZWFmNjE5Y2RkM2MvZTgzYjJhOTEtOTNkNC00MjNhLTllYWItNDg4YzY0NGY1NzdiIiwiaWF0IjoxNzA1NzM2NjcxLCJleHAiOjMzMjc2Mjk2NjcxfQ.8_api5K6YbQe3LbQXAfP03BvOQBMzX7kCM5Iqfbj6EE "=123x")

In Google Chrome it works like it should.

### Expected Behavior

I would expect this just to be transparent in all browsers.

### Steps to Reproduce

1. create a component with rounded corners for the root container
2. use this component as a Custom Dialog that opens on an action.
3. Check the result in Safari for Mac.

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

ITFfhcj15ZN2xNtD14SJbvowq2MsHGkcTL4zlO1HcxAYGO/yPbJ/fMjRZRdvZPXkYWlhHFGJgmEK/+zLi+zLCPApFxacfqJGwpFQZDqXQXq8SrKaEbWKPGhPL9ljClSn0re42h5NLtVuZVIX2ly6H9iWRXTBJ5+/Zwh9f6PHaOI=

### Context

Because of this issue I cannot make the component look the way I want according to my companies corporate identity. The only way I could see myself fixing this is not using rounded corners on the Custom Dialog popups, but that's not the way the app is designed.

### Visual documentation

[https://github.com/FlutterFlow/flutterflow-issues/assets/156412703/a9c75df3-e4da-4f39-8e2f-81d0d44468e5](https://github.com/FlutterFlow/flutterflow-issues/assets/156412703/a9c75df3-e4da-4f39-8e2f-81d0d44468e5)

### Additional Info

*No response*

### Environment

```
- FlutterFlow version: 4.0.3
- Platform: Flutterflow Mac
- Browser name and version: Safari 17.2.1
- Operating system and version affected: Mac OSx Sonoma 14.2.1
```

**Comment by JP-BLURP** [comment]
I've been experience the same thing

**Comment by sbis04** [comment]
I couldn't reproduce the issue on my side when I tested it on a demo project. But can you provide a simple project with just the issue setup? (that will help us to debug faster)

**Comment by DarckETH** [comment]
> I couldn't reproduce the issue on my side when I tested it on a demo project. But can you provide a simple project with just the issue setup? (that will help us to debug faster)
> 
> Also, does this issue happen when you run the app in the test mode?

Hello, I have just made a new demo app, and added the containers where needed, and it is here as well, here is the link to the app;
https://blank-vx3zhj.flutterflow.app

Are you sure you are on the same browser/version? Since in chrome it's not there, but in Safari it is.

For Testmode this is the same, Chrome no issue, Safari the dark corners. It really seems like on safari it has some sort of default corners and background that's et for the main root element. Since it also has a small corner radius. (smaller than the 20px that I set for my rounded corners.

<img width="87" alt="Scherm­afbeelding 2024-01-15 om 07 56 37" src="https://github.com/FlutterFlow/flutterflow-issues/assets/156412703/2d7dd99a-e1f7-405f-9267-3f63f85fbff6">
